resources/views/contests/show.blade.php

```diff
@@ -27,6 +27,9 @@
                     <li>
                         <a href="{{ route('split', $contest->waitingRoom->id) }}" class="button -secondary">Split</a>
                     </li>
+                    <li>
+                        <a href="{{ route('contests.user.add', $contest->id) }}" class="button -secondary">+ User</a>
```

Comment:
Maybe call the button "Signup User" since we already have a "Add User" in the user crud to avoid confusion?